esp-wifi/MIGRATING-0.9.md

```diff
@@ -39,3 +37,26 @@ You no longer have to set up clocks and pass them to `esp_wifi::initialize`.
      // ...
  }
 ```
+
+## Memory allocation
```

Comment:
We should also mention how to use another allocator here, that they'll need to provide those alloc_internal and free mem functions themselves